### PR TITLE
Quest 3 completed.

### PR DESCRIPTION
**Pull request by snehasharma76**

### CODE REVIEW

File: sources/quest_3/chapter_10/step_28/step_28_start.move [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+       let currentID = info.id;
+       let nextID: u64 = currentID + 1;
+       let _info = VisitorInfo<u64, u64> {
+            id: nextID,
+            age: visitor_age,
+        };
+    }
+
+    //create a public(friend) function `daily_visitors()` of type u64 that returns the value `6`
+    //create a module named `tour_guide` under the package `robinson`
```
Comment (0xNekr) [inline sources/quest_3/chapter_10/step_28/step_28_start.move:179]: This should be after the `}`, instructions should goes to place user have to use

File: sources/quest_3/chapter_10/step_28/step_28_target.move [hunk 2]
```diff
@@ -0,0 +1,193 @@
+module robinson::my_shore {
+friend robinson::tour_guide;
```
Comment (0xNekr) [inline sources/quest_3/chapter_10/step_28/step_28_target.move]: A little tab to keep it in line

**Review by 0xNekr — CHANGES_REQUESTED** [review]

File: sources/quest_3/chapter_10/step_28/step_28_target.move (hunk 2 above)
Comment (snehasharma76) [reply sources/quest_3/chapter_10/step_28/step_28_target.move]: done

File: sources/quest_3/chapter_10/step_28/step_28_start.move (hunk 1 above)
Comment (snehasharma76) [reply sources/quest_3/chapter_10/step_28/step_28_start.move:179]: done

**Review by 0xNekr — APPROVED** [review]

**Pull request merged**